eolearn/core/eoexecution.py

```diff
@@ -55,6 +55,7 @@ class _ProcessingData:
     filter_logs_by_thread: bool
     logs_filter: Filter | None
     logs_handler_factory: _HandlerFactoryType
+    temporal_dim_warning_is_error: bool
```

Comment:
I'm having doubts about the name, but not sure about alternatives. Maybe:
- `raise_on_temporal_mismatch`
- `temporal_mismatch_is_error`

or something along those lines

Reply:
yeah,  no matter what you try you end up with something long :see_no_evil: I was thinking of shortening it to `raise_on_tdim_missmatch` but `tdim` is kinda  non-standard

Reply:
Yes, generally I meant more for the parameter in the function than the private one. Perhaps I'm more inclined to use the one with `raise` because I remember seeing it in libraries a few times as a parameter as well.

but in the end it's a specific thing, which requires a specific name, so if you don't like the suggestions, it's fine and feel free to leave it as it is.